worldeditadditions_farwand/lib/cloudwand.lua

```diff
@@ -1,52 +1,22 @@
-local function add_point(name, pos)
-	if pos ~= nil then
-		-- print("[set_pos1]", name, "("..pos.x..", "..pos.y..", "..pos.z..")")
-		if not worldedit.pos1[name] then worldedit.pos1[name] = vector.new(pos) end
-		if not worldedit.pos2[name] then worldedit.pos2[name] = vector.new(pos) end
-		
-		worldedit.marker_update(name)
-		
-		local volume_before = worldedit.volume(worldedit.pos1[name], worldedit.pos2[name])
-		
-		worldedit.pos1[name], worldedit.pos2[name] = worldeditadditions.vector.expand_region(worldedit.pos1[name], worldedit.pos2[name], pos)
-		
-		local volume_after = worldedit.volume(worldedit.pos1[name], worldedit.pos2[name])
-		
-		local volume_difference = volume_after - volume_before
-		
-		worldedit.marker_update(name)
-		worldedit.player_notify(name, "Expanded region by "..volume_difference.." nodes")
-	else
-		worldedit.player_notify(name, "Error: Too far away (try raising your maxdist with //farwand maxdist <number>)")
-		-- print("[set_pos1]", name, "nil")
-	end
-end
-local function clear_points(name, pos)
-	worldedit.pos1[name] = nil
-	worldedit.pos2[name] = nil
-	worldedit.marker_update(name)
-	worldedit.set_pos[name] = nil
-	
-	worldedit.player_notify(name, "Region cleared")
-end
+local wea = worldeditadditions
 
 minetest.register_tool(":worldeditadditions:cloudwand", {
-	description = "WorldEditAdditions far-reaching point cloud wand",
+	description = "WorldEditAdditions far-reaching point cloud selection wand",
```

Comment:
My fault for not describing it better, but while we're here this description works better I think:
```suggestion
	description = "WorldEditAdditions far-reaching additive point cloud selection wand",
```

Reply:
Well now it's too wordy I think

Reply:
we could call it the selection addition wand or the additive selector wand.

Reply:
Hrm, good point. What about this then?
```suggestion
	description = "WorldEditAdditions far-reaching additive selectior wand",
```